### PR TITLE
feat: create a rook-ceph cluster

### PR DESCRIPTION
**Pull request by Javlopez**
full cycle is complete, now this can create completely and async a rook-ceph cluster

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Koor-Tech Genesis
+
+
+Welcome to koor Genesis
+
+Our stack
+- Go
+- RabbitMQ
+- Postgres
+
```
Comment (galexrt) [inline README.md:10]: Let's add a section talking about the license the code is under, and let's add a license file.

File: pkg/ssh/ssh.go [hunk 2]
```diff
@@ -86,60 +86,3 @@ func RunAgent(sshPrivateKey string) error {
 	}
 	return nil
 }
-
-/*
```
Comment (galexrt) [inline pkg/ssh/ssh.go:90]: If we don't need this code, let's remove it altogether. git has the code if we need it back.

**Review by galexrt — COMMENTED** [review]
My main question would be how do we wanna authenticate users here?
Or will there be a separate "frontend" service that takes care of the authentication?

File: README.md (hunk 1 above)
Comment (dave-at-koor) [reply README.md:10]: Which license should we use? MIT, or something more restrictive?

**Review by dave-at-koor — APPROVED** [review]
This is a good start.

**Comment by dave-at-koor** [comment]
> My main question would be how do we wanna authenticate users here? Or will there be a separate "frontend" service that takes care of the authentication?

Great question to answer as we review the system design. https://docs.google.com/document/d/1Qp6n57Y1Jsw-FTpgp4Nnw6PrvlCVvDrPi53cEQEBd94/edit#heading=h.4jkujba0ssx

File: README.md (hunk 1 above)
Comment (galexrt) [reply README.md:10]: Let's go with Apache 2.0 license. I have opened a PR #2 to add it.

**Pull request merged**